### PR TITLE
fix the wrong word

### PR DESCRIPTION
**Pull request by tanjunchen**
@nicolasnoble

### CODE REVIEW

**Comment by tanjunchen** [comment]
@lidizheng

File: tools/profiling/microbenchmarks/bm_diff/bm_main.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def _args():
         '--pr_comment_name',
         type=str,
         default="microbenchmarks",
-        help='Name that Jenkins will use to commen on the PR')
+        help='Name that Jenkins will use to commenton the PR')
```
Comment (lidizheng) [inline tools/profiling/microbenchmarks/bm_diff/bm_main.py]: comment on

Comment (tanjunchen) [reply tools/profiling/microbenchmarks/bm_diff/bm_main.py]: Thank You I Will Change It

**Comment by tanjunchen** [comment]
@lidizheng

**Comment by lidizheng** [comment]
Known failure: https://github.com/grpc/grpc/issues/20421 https://github.com/grpc/grpc/issues/20425

**Pull request merged**